src/main/java/core/basesyntax/service/impl/TransactionsServiceImpl.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FruitTransactionParser;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class TransactionsServiceImpl implements FruitTransactionParser {
+    private static final int OPERATION_INDEX = 0;
+    private static final int FRUIT_INDEX = 1;
+    private static final int AMOUNT_INDEX = 2;
+    private static final String LINE_SEPARATOR = ",";
+
+    @Override
+    public List<FruitTransaction> parse(String data) {
+        String[] lines = data.split(System.lineSeparator());
+        return Arrays.stream(lines)
+                .skip(1)
```

Comment:
Why with you skip 1 line?  Let's describe it in the service name. For example, it can be named CsvFileParser.
But better to remove the first line int the CsvFileReader

Reply:
fixed with constant